docs/examples/kops-tests-private-net-bastion-host.md

```diff
@@ -148,8 +148,8 @@ kind: InstanceGroup
 metadata:
   name: bastions
 spec:
-  image: kope.io/k8s-1.7-debian-jessie-amd64-hvm-ebs-2017-07-28
-  machineType: t2.micro
+  image: 099720109477/ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-20200907
```

Comment:
There is another reference to a `kope.io` image in the file. Please update that one too.
Also, please change `t2.micro` to `t3.micro` everywhere.

Reply:
I changed it in https://github.com/kubernetes/kops/pull/10044/commits/aff8c0af0090bb8e61c9c11a55c418547b95082b